### PR TITLE
Initial governance

### PR DESCRIPTION
**Pull request by canesin**
Please @CityOfZion/council review and iterate before we accept.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
-# governance
-Home of Zion rules and reports
+# Community governance
+
+This is home of City of Zion (CoZ) community rules and reports.
+Our formalization is the reflect of a humble move from NEO team to receive our contributions and if quality work is continued provide support.
+
+There is complete freedom in the targets and objectives off the community. That said, our goal is to improve NEO, so we will align with needs and avoid re-work as much as possible, unless it is to attain a new level of quality.
```
Comment (AshRolls) [inline README.md]: Small typo: [..] and objectives **of** the community

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+
+7 - Rewards distribution will be voted by project, before being received. There must be majority agreement of all 9 council members.
+
+8 - All decisions not explicitly treated in this list will be voted as item 7.
+
+9 - Item 7 voting polls of the council will run for 1 week on #council slack channel and after closing published in the #dev channel.
+
+10 - The high council will be voted every quarter, we will sync with years quarters. The first election will be on 1 October 2017. The newly elected high council can propose changes to this rules during the 1st week after the election, changes should be voted as item 7.
+
+11 - Any member of the council or high council is free to give-up the position at any time, when this happen a election on the council will be made and majority must be achieved to design a replacement.
```
Comment (AshRolls) [inline README.md]: Small error: [..] achieved to **decide** a replacement.

**Comment by canesin** [comment]
2 months ? I have 45 days vacations.

File: README.md (hunk 1 above)
Comment (anfn101) [inline README.md]: More edits: 
"This is **the** home of"
"**the** City of Zion"
"is **to reflect a humble**"
"from **the** NEO team"
"align with **their** needs"
"avoid **reworking**" 
"**higher level of quality"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
-# governance
-Home of Zion rules and reports
+# Community governance
+
+This is home of City of Zion (CoZ) community rules and reports.
+Our formalization is the reflect of a humble move from NEO team to receive our contributions and if quality work is continued provide support.
```
Comment (AshRolls) [inline README.md]: Our formalization is **a reflection** of

File: README.md (hunk 2 above)
Comment (anfn101) [inline README.md]: More edits: 
"will be **elected**" every quarter. **These elections will coincide with each quarter of the calendar year.**"
"to **these** rules"
"**first** week after the election. Changes"
"**give up their**"
"at any time**. W**hen this **happens**, **an** election on the council will be **held** and **a** majority must be achieved to **select** a replacement."

File: README.md (hunk 3 above)
Comment (anfn101) [inline README.md]: I like "Our formalization  **is to reflect a humble**... but either one is fine. Canesin can choose.

**Comment by localhuman** [comment]
2 months is good! I just grabbed 1 month out of the air.

**Comment by anfn101** [comment]
You could also just go with 3 months for a quarter year... but by then we'd be electing someone new anyway (to the high council). So this would really just apply to the council as a whole...

**Comment by localhuman** [comment]
Just trying to avoid a situation where we have 4 to 4 votes, with 1 person not responding.

**Comment by canesin** [comment]
OK, just added observations..

**Comment by anfn101** [comment]
A 4 to 4 vote does seem likely to happen at some point... but then 1 or 2 months might be too long. We should come up with a way to resolve this issue if it occurs.

**Comment by anfn101** [comment]
In any case, this looks good for a first version. I'll branch and push one later with some light editing, so that Canesin doesn't have to handle it. If we have anything else to add to it we can also do that later.

**Pull request merged**